### DISCUSSION TITLE
[Question][Hw3] What should be defined and what is given

### DISCUSSION THREAD

**Discussion by m-spitfire** [prosyslab-classroom/cs524-program-analysis#297] [Q&A]
Name: Murad Bashirov

Can we use $\gamma_{\mathbb{Z}}$, $\alpha_{\mathbb{Z}}$, and $+^\sharp$ without defining them (since they're defined in the lecture) in Problem 6 or do we have to write their definitions as well?

**Comment by goodtaeeun** [comment]
I am sorry for the late reply.
Please define what you are using.

**Comment by m-spitfire** [comment]
Thanks for the reply. I hope for next assignments you could answer in time.
